test/integration/spec/rtcpeerconnection.js

```diff
@@ -865,7 +865,8 @@ function testRemoveTrack() {
         }
       });
 
-      it(`should ${shouldThrow ? '' : 'not '}throw`, () => {
+      // NOTE(mhuynh): Expected to fail in Chrome.
+      (isChrome ? it.skip : it)(`should ${shouldThrow ? '' : 'not '}throw`, () => {
```

Comment:
@mhuynh5757 I think we should make sure that `PeerConnection#removeTrack` should not throw an exception here, instead of skipping the test.

Reply:
I checked two things:

* Based on [this usage here](https://github.com/twilio/twilio-video.js/blob/1.11.1/lib/signaling/v2/peerconnection.js#L819), twilio-video.js should be immune to throws, since we check `_rtpSenders` first.
* Based on the spec, this should not throw. The `removeTrack` algorithm says

  > 4. If _sender_ was not created by _connection_, throw an `InvalidAccessError`.
  > 5. Let _senders_ be the result of executing the `CollectSenders` algorithm.
  > 6. If _sender_ is not in _senders_ (which indicates that it was removed due to setting an RTCSessionDescription of type "rollback"), then abort these steps.

I believe that Chrome's behavior is incorrect here. Step 4 should not throw the error it throws (@mhuynh5757 do you mind actually sharing the exact error?). Instead, we should continue to Step 6 and abort. There's actually probably a Chrome/WebRTC bug worth filing here :-)

Reply:
^ All this is to say, I agree with @mhuynh5757 — let's treat this is a Chrome bug, open a Chrome/WebRTC issue, do a `try`/`catch` in `removeTrack`, and un-`skip` the test.

Reply:
The exact error that is thrown is a DOMException... The sender was not created by this peer connection. I agree, this sounds like a Chrome bug, which also happens on Canary. Will amend commit according to @markandrus.